OneCommunityGlobal/HighestGoodNetworkApp
# Nahiyan_Optimize dashboard for mobile display

# Description
This PR makes the dashboard 100% compatible with mobile displays (375px width and up).

## Related PRS (if any):
This frontend PR is related to the dev backend

## Main changes explained:
- Updated fonts on all components so they become more responsive.
- Updated some icons to be more responsive at smaller viewports
- Added max height on leaderboard and timelog tasks as they can take up too much space.
- Fixed widths on some components.
- Leaderboard will now show up at the bottom when at smaller viewports.

## How to test:
1. check into the current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. Clear site data/cache
4. log as any user
5. go to the dashboard
6. verify that all components and inner components on the dashboard are working properly and have no visual issues at all viewports.
7. verify this new feature works in [dark mode](https://docs.google.com/document/d/11OXJfBBedK6vV-XvqWR8A9lOH0BsfnaHx01h1NZZXfI)
8. The HEADER is NOT part of the PR.

## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/71025942/5877ccb8-7be6-4e46-8c40-71129822e431
